File: priv/javascript/lib/atom.js
import Erlang from './erlang';

let Atom = {};

Atom.__MODULE__ = Erlang.atom("Atom");

Atom.to_string = function (atom) {
  return Symbol.keyFor(atom);
};

Atom.to_char_list = function (atom) {
  return Atom.to_string(atom).split('');
};

export default Atom;

File: priv/std_lib/atom.ex
defmodule ElixirScript.Atom do
   @moduledoc false 
  import Kernel, except: [to_string: 1]

  def to_char_list(atom) do
    to_string(atom).split("")
  end

  def to_string(atom) do
    Symbol.keyFor(atom)
  end

end
